src/main/kotlin/app/revanced/patches/all/ads/blocklist/AndroidAdsBlocklist.kt

```diff
@@ -0,0 +1,31 @@
+package app.revanced.patches.all.ads.blocklist
+
+import org.jf.dexlib2.Opcode
+
+val blockUrls: List<String> = listOf(
+    "adservice.google.com",
+    "g.doubleclick.net",
+    "googleadservices.com",
+    "amazon-adsystem.com",
+)
+
+val replaceUrlsWith: String = "https://example.com"
+
+val stringOpcodes: List<Opcode> = listOf(
+    Opcode.CONST_STRING
+)
+
+val blockInvokes: List<String> = listOf(
+    "loadAd",
+    "AdListener",
+    "AdLoad",
+    "AdView",
+)
```

Comment:
A comment explaining how you got the list of method names or what they correspond to would be nice. For example "Google ads" or "Amazon ads" etc.